include/seccomp.h.in

```diff
@@ -94,6 +94,9 @@ enum scmp_compare {
 	SCMP_CMP_GT = 6,		/**< greater than */
 	SCMP_CMP_MASKED_EQ = 7,		/**< masked equality */
 	_SCMP_CMP_MAX,
+
+	SCMP_CMP_OPMASK = 0xFFFF,
+	SCMP_CMP_32BIT = 1 << 16,	/**< operation is 32-bit */
```

Comment:
I'm still working my way through this PR, but I think we want to move `SCMP_CMP_OPMASK` and `SCMP_CMP_32BIT` out of the enum and have them be standalone preprocessor macros/constants.

Reply:
I guess if we do that we need to add a "filler" value to cause the enum to be a certain size, which feels a little icky but I think it's okay and if make sure to represent all of the flag space, e.g.
```
enum scmp_compare {
	_SCMP_CMP_MIN = 0,
	SCMP_CMP_NE = 1,
	SCMP_CMP_LT = 2,
	SCMP_CMP_LE = 3,
	SCMP_CMP_EQ = 4,
	SCMP_CMP_GE = 5,
	SCMP_CMP_GT = 6,
	SCMP_CMP_MASKED_EQ = 7,
	_SCMP_CMP_MAX,
	_SCMP_CMP_OPMASK = 0x0000FFFF,
	_SCMP_CMP_FLAGMASK = 0xFFFF0000,
};
```

Reply:
... which somewhat questions the value of moving the flag definitions out of the enum and into their own macros/constants, but that still seems like a goodish idea to me at this point.